rotkehlchen/chain/ethereum/modules/l2/loopring.py

```diff
@@ -253,11 +257,14 @@ def _api_query(
                 result_info = json_ret.get('resultInfo', None)
                 if result_info:
                     code = result_info.get('code', None)
-                    # This two codes can appear if there is an error with the
-                    # key as tested here
-                    # https://github.com/rotki/rotki/pull/2569#issuecomment-800151821
-                    if code and code in (100001, 101002):
-                        raise LoopringAPIKeyMismatch()
+                    # From the documentation the code returned when the ApiKey
+                    # is not valid is 104002 https://docs3.loopring.io/en/?q=104002
+                    if code and code == 104002:
```

Comment:
improvement/nitpick:

Here you are checking that `code` is not `None` 2 times.

You can:

```python
    if code is not None:
        if code == 104002:
            raise LoopringInvalidApiKey()
        elif code == 101002:
            raise LoopringUserNotFound()
```
```